Lib/test/test_asyncio/test_streams.py

```diff
@@ -1038,15 +991,10 @@ async def inner(httpd):
 
         self.assertEqual(messages, [])
 
-    def test_eof_feed_when_closing_writer(self):
+    async def test_eof_feed_when_closing_writer(self):
         # See http://bugs.python.org/issue35065
-        messages = []
-        self.loop.set_exception_handler(lambda loop, ctx: messages.append(ctx))
```

Comment:
Right. Actually, I've removed all usages of `messages` inter-thread communication list:

1. ```diff
   -        messages = []
   -        self.loop.set_exception_handler(lambda loop, ctx: messages.append(ctx))
   ```
1. ```diff
   -        self.assertEqual(messages, [])
   ```

because the server is now async, lives in a main thread, and needs no intermediary to pass exceptions into a test harness.

Reply:
Yes, I've removed all usages of `messages` inter-thread communication list:

1. ```diff
   -        messages = []
   -        self.loop.set_exception_handler(lambda loop, ctx: messages.append(ctx))
   ```
1. ```diff
   -        self.assertEqual(messages, [])
   ```

because the server is now async, lives in a main thread, and needs no intermediary to pass exceptions into a test harness.

Reply:
By the way, thank you for noticing, I've added the clarification as the second paragraph of the first PR comment.